src/components/Card/index.js

```diff
@@ -26,30 +24,36 @@ export default class Card extends React.Component {
     return JobService.getPreviewJobID(id)
   }
 
+  huegenerator = () => {
+    return 'rgb(' + (Math.floor((205)*Math.random())+50) + ',' + (Math.floor((205)*Math.random())+50) + ',' + (Math.floor((205)*Math.random())+50) + ')';
+  }
+
   componentDidMount(){
       $('.card').each(function () {
-          var hue = 'rgb(' + (Math.floor((205)*Math.random())+50) + ',' + (Math.floor((205)*Math.random())+50) + ',' + (Math.floor((205)*Math.random())+50) + ')';
-          $(this).css("border-top-color", hue);
+          $(this).css("border-top-color", this.huegenerator());
```

Comment:
This breaks the component since on line 33, `this` now refers to the selected card component and not the react version of `this` that you are presuming this to be, can be fixed as follows:
```
let huegenerator = this.huegenerator;
$('.card').each(function () {
    $(this).css("border-top-color", huegenerator());
});
```